scripts/reset_wall.sh

```
@@ -4,22 +4,22 @@
 set -e
 
 VOLUME="$1"
-VERSION="$2"
+IMAGE="$2"
 
 # Input validation
 if [ -z $VOLUME ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)